llama-index-core/llama_index/core/workflow/workflow.py

```diff
@@ -394,6 +407,26 @@ async def _run_workflow() -> None:
         asyncio.create_task(_run_workflow())
         return result
 
+    @dispatcher.span
+    def run_from(
+        self, ctx: Optional[Context], checkpoint: Checkpoint, **kwargs: Any
```

Comment:
small nit about UX, we could move `ctx` after the checkpoint and default to `None` so users don't need to care at all when not using it (rather than having to pass `ctx=None`)

Reply:
good call!